test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Kindly note that this file is supposed to be ignored by the .gitignore file. To make sure of this, delete the file and commit it. then add it again and when you push it to GitHub, it will not be visible online but only on your local version. Try removing the forward slash before your test,md file in your .gitignore first.

Essentially, you want is to commit the .gitignore with the name of the file you want to ignore before you add and commit that file that you want to be ignored.

Reply:
test.md file delete from the online repo, but still exists in local directory due to .gitignore file